spec/factories.rb

```diff
@@ -385,28 +386,28 @@ def r_str
   factory :report do
     user
     association :item, factory: :status_message
-    text "offensive content"
+    text { "offensive content" }
   end
 
   factory :o_auth_application, class: Api::OpenidConnect::OAuthApplication do
     client_name { "Diaspora Test Client #{r_str}" }
-    redirect_uris %w(http://localhost:3000/)
+    redirect_uris { %w(http://localhost:3000/) }
   end
 
   factory :o_auth_application_with_ppid, parent: :o_auth_application do
-    ppid true
-    sector_identifier_uri "https://example.com/uri"
+    ppid { true }
+    sector_identifier_uri { "https://example.com/uri" }
   end
 
   factory :o_auth_application_with_xss, class: Api::OpenidConnect::OAuthApplication do
-    client_name "<script>alert(0);</script>"
-    redirect_uris %w(http://localhost:3000/)
+    client_name { "<script>alert(0);</script>" }
+    redirect_uris { %w(http://localhost:3000/) }
   end
 
   factory :auth_with_read, class: Api::OpenidConnect::Authorization do
     o_auth_application
     user
-    scopes %w(openid sub aud profile picture nickname name read)
+    scopes { %w(openid sub aud profile picture nickname name read) }
```

Comment:
Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`.